assets/components/molecules/nav/nav.scss

```diff
@@ -27,7 +27,7 @@ $nav-spacer: 4px;
     left: 100%;
     width: 100%;
     margin: 0;
-    padding: 0 2rem 0 3rem;
+    padding: 0 2 * $spacer 0 3 * $spacer;
```

Comment:
I prefer when values are wrapped in parenthesis, it's easier to read (even if mathematically not needed).

```scss
padding: 0 (2 * $spacer) 0 (3 * $spacer);
```

Reply:
nice idea!